cogs/basic_command.py

```diff
@@ -22,15 +22,17 @@ async def bothelp(self, ctx):
                 'can be one of \'all\', \'troop\', \'spell\' or \'building\'. Defaults to \'all\' if not specified',
                 f'[!ausclanboat]({url}) - Lists clan members who attacked enemy boats this week.',
                 f'[!ausclanwar [weeks]]({url}) - Shows the war peformance of the players in x amount of weeks. Defaults to 4 weeks if not specified',
-                f'[!ausclanremaining]({url}) - Shows the remaining war attacks & remaining players count for the current river race'
+                f'[!ausclanremaining]({url}) - Shows the remaining war attacks & remaining players count for the current river race',
+                f'[!ausclanwarplayers]({url}) - Shows all players within the clan and players who left the clan who has not completed all war attacks'
             ]
             generic_cmds = [
                 f'[!bothelp]({url}) - Shows this help message about available commands.',
                 f'[!membercardsranked [clan_tag] [filter]]({url}) - Does same thing as `!ausclan` but '
                 'for the specific [clan_tag] instead.',
                 f'[!boatattack [clan_tag]]({url}) - Does same thing as `!ausclanboat` but for the specific [clan_tag] instead.',
                 f'[!riverwar [clan_tag] [weeks]]({url}) - Does same thing as `!ausclanwar` but for the specific [clan_tag] instead.',
-                f'[!clanremaining [clan_tag]]({url}) - Does same thing as `!ausclanremaining` but for the specific [clan_tag] instead.'
+                f'[!clanremaining [clan_tag]]({url}) - Does same thing as `!ausclanremaining` but for the specific [clan_tag] instead.',
+                f'[!currentwarplayers]({url}) - Does same thing as `!ausclanwarplayers` but for the specific [clan_tag] instead.'
```

Comment:
Change this to `clanremainingplayers` after above comment rename too

Reply:
done